ports/raspberrypi/boards/challenger_rp2040_subghz/mpconfigboard.mk

```diff
@@ -0,0 +1,13 @@
+USB_VID = 0x2e8a
+USB_PID = 0x1030
```

Comment:
This doesn't match what I see here: https://github.com/raspberrypi/usb-pid/blob/main/Readme.md